react-app/src/components/Result.js

```diff
@@ -1,7 +1,45 @@
 import React from 'react';
 import { Accordion } from 'react-bootstrap';
 
+const _ = require('lodash');
+const fhirpath = require('fhirpath');
+
 function Result(props) {
+  function countResources() {
+    const resourceList = [];
+    // iterate through resources array, checking the resourceType of each entry
+    // if (new type),
+
+    const allResourceTypesPath = 'Bundle.descendants().resource.resourceType';
+    const uniqueResourceTypes = _.uniq(fhirpath.evaluate(props.bundle, allResourceTypesPath));
+
+    const countThisResource = (resourceType) =>
+      `Bundle.descendants().where(resource.resourceType = '${resourceType}').count()`;
+    uniqueResourceTypes.forEach((resourceType) => {
+      const count = fhirpath.evaluate(props.bundle, countThisResource(resourceType));
+      resourceList.push({ resourceType, count });
+    });
+
+    return _.sortBy(resourceList, ['resourceType', 'count']);
+  }
+
+  function getResourceList() {
+    const resourceList = countResources();
+    return resourceList.map((item, i) => (
+      <p key={i}>
```

Comment:
DP3 - It's discouraged to use index-in-an-array as a key for a `.map`. The reasons for it being less-than-ideal won't materialize here, but we should really be using something that is unique to the thing being rendered in the iteration of the loop (for example, the resourceType!). If you want to read more look here: https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318